src/nvim/lua/vim.lua

```diff
@@ -1,3 +1,65 @@
+local function system(cmd)
```

Comment:
Until we have lua native job control this should probably use `systemlist` (or even better viml job control to distinguish stderr) internally and do a safe conversion to lua binary string, so that this function handles binary data correctly (unlike vimL `system()` where this is impossible)

Reply:
or even remove the function to force using systemlist which is better in every way.

Reply:
@teto this is not really true in Lua, the issue is just that vimL system() should not be used to define Lua system()